ex10_ch3.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#create constant
+SALES_TAX = 5.5
+
+#step1: get input for price of item 1
+item1_price = int(input("Enter the price of item 1: "))
+
+#step2: get input for quantity of item 1
+item1_quantity = int(input("Enter the quantity of item 1: "))
+
```

Comment:
Here's an example of a time that I'd like to use a line break and the "/n" shows up. For example: 
item1_quantity = int(input("Enter the quantity of item 1: /n")) ---> the "/n" shows up.

How do I insert the line break without /n showing up?

Reply:
Hmmm, that works correctly on my machine. I'll have to look at that on yours.